src/main/java/SuBOT.java

```diff
@@ -0,0 +1,73 @@
+import java.util.Scanner;
+
+public class SuBOT {
+    private static boolean stopFlag = false;
+    private static Task[] tasks = new Task[100];
```

Comment:
Is it wise to use magic numbers? Perhaps you could make it a final variable.

Reply:
Thank you for the suggestion! Added! 👍